aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -180,9 +215,7 @@ export const SignUpExample = () => {
                             <FormCheckmark size="small" />
                           </Box>
                         ) : (
-                          <Box alignSelf="center">
-                            <FormClose size="small" />
-                          </Box>
+                          <></>
```

Comment:
instead of doing an empty tag, can we change line 213 to be:
```
formValues.password && rule.valid &&
```
then we don't have to do a `?` and `:`